espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/helpers/ActivityOptionsHelpers.kt

```diff
@@ -0,0 +1,71 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.appium.espressoserver.lib.helpers
+
+import android.app.ActivityOptions
+import kotlin.reflect.KClass
+import kotlin.reflect.full.cast
+
+private fun <T : Any> requireType(expectedType: KClass<T>, key: String, value: Any?): T {
```

Comment:
Please do not copypaste existing helpers. Simply move it to a common module, so they can be reused

Reply:
yes. You are right, i found i don't need to copypaste again as they are in the same package